### ISSUE TITLE
Encargar "entradas"

### ISSUE THREAD

**Issue by etchegaray** [olvidio/tramity#39]
Cuando un director encarga una entrada a un oficial facilitaría que pudiera 'decirle' algo del estilo: "para enviar a los ctr", "no lo envíes todavía", "estudialo y hablamos", "prepara un borrador para la oficina", etc.

**Comment by Jordi59** [comment]
Sí. Es algo que también nos lo reclaman aquí. De entrada no lo hicimos por la complejidad de añadir un campo que no es propiamente de una entrada del registro, sino que se queda en el ámbito de la oficina. Podría ser un campo que solo fuera visible para la oficina. El problema está en que hay escritos con varias oficinas, y si el d de otra oficina (no principal) también quiere encargar a un of el escrito para contestar su parte y quiere poner un comentario, ha de ser otro campo distinto. Debería haber tantos campos de "comentario" como oficinas, y esto genera complejidad. Pero interesaría resolverlo.
